user_interface.py

```diff
@@ -1,157 +0,0 @@
-"""Welcome to Amity Room Allocation App
-Usage:
-  create_room <room_type> <room_name>...
-  add_person <fname> <lname> <role> [--wants_accomodation=N] 
-  print_allocations [--o=filename]
-  print_unallocated [--o=filename]
-  load_people <filename>
-  reallocate_person <person_name> <new_room_name>
-  print_room <room_name>
-  save_state [--db=sqlite_database]
-  load_state <sqlite_database>
-
-Options:
-  -h --help     Show this screen.
-  --version     Show version.
-"""
-
-from docopt import docopt, DocoptExit
-from termcolor import cprint, colored
-from pyfiglet import figlet_format
-import cmd
-import sys
-import os
-
-from app.classes.amity import Amity
-
-amity = Amity()
-
-def docopt_cmd(func):
-    """
-    This decorator is used to simplify the try/except block and pass the result
-    of the docopt parsing to the called action.
-    """
-    
-    def fn(self, arg):
-        try:
-            ARGS = docopt(fn.__doc__, arg)
-
-        except DocoptExit as e:
-            # The DocoptExit is thrown when the args do not match.
-            # We print a message to the user and the usage block.
-            print('Invalid Command!')
-            print(e)
-            return
-
-        except SystemExit:
-            # The SystemExit exception prints the usage for --help
-            return
-
-        return func(self, ARGS)
-
-    fn.__name__ = func.__name__
-    fn.__doc__ = func.__doc__
-    fn.__dict__.update(func.__dict__)
-    return fn
-
-
-class AmityApp(cmd.Cmd):
-
-
-
-    cprint(figlet_format('AMITY APP', font='doom'), 'magenta')
-    cprint('*' * 60, 'white')
-    cprint("    AMITY ROOM ALLOCATION SYSTEM.    ", 'cyan')
-    cprint('*' * 60, 'white')
-    cprint(" Below is a list of available commands.", 'cyan')
-    cprint('*' * 60, 'white')
-    cprint(__doc__, 'cyan')
-
-
-    @docopt_cmd
-    def do_create_room(self, arg):
-        """ Usage: create_room <room_type> <room_name>..."""
-        for name in arg['<room_name>']:
-            amity.create_room(name, arg['<room_type>'])
-
-    @docopt_cmd
-    def do_add_person(self, arg):
-        """ Usage: add_person <fname> <lname> <role> [--wants_accomodation=N]
-
-        Options:
-        -w, --wants_accomodation=<N>  Wants accomodation [default: N]
-        """
-        person_name = arg["<fname>"].upper() + " " + arg["<lname>"].upper()
-        fname =arg['<fname>'] 
```

Comment:
maybe stop being lazy and name the variables more clearly. i.e fname - firstname and so on.